src/cheri_vmem_types.sail

```diff
@@ -10,13 +22,15 @@ let default_write_acc : ext_access_type = Data
 val accessType_to_str : AccessType(ext_access_type) -> string
 function accessType_to_str (a) =
   match (a) {
-    Read(Data)      => "R",
-    Write(Data)     => "W",
-    ReadWrite(Data) => "RW",
-    Read(Cap)       => "RC",
-    Write(Cap)      => "WC",
-    ReadWrite(Cap)  => "RWC",
-    Execute()       => "X"
+    Read(Data)           => "R",
+    Write(Data)          => "W",
+    ReadWrite(Data,Data) => "RW",
+    Read(Cap)            => "L",
+    Write(Cap)           => "S",
+    ReadWrite(Cap,Cap)   => "LS",
+    ReadWrite(Data,Cap)  => "RS",
+    ReadWrite(Cap,Data)  => "LW",
```

Comment:
Leave these as they were (well, extended to disambiguate the ReadWrite cases by putting an extra C after the R when appropriate)? The abbreviations no longer match up with the names.